clients/py/v3io_frames/client.py

```diff
@@ -190,23 +190,29 @@ def delete(self, backend, table, filter='', start='', end='',
         backend : str
             Backend name
         table : str
-            Table to create
+            Table or stream to delete or from which to delete specific items
         filter : str
-            Filter for selective delete
-        start : string
-             (`tsdb` backend only) Start (minimum) metric-sample time for the
+            (`kv` backend only) A filter expression that identifies specific
+            items to delete; for example, "age<18"
+        start : str
+             (`tsdb` backend only) Start (minimum) data sample time for the
              delete operation, as a string containing an RFC 3339 time, a Unix
              timestamp in milliseconds, a relative time (`"now"` or
              `"now-[0-9]+[mhd]"`, where `m` = minutes, `h` = hours, and `'d'` =
-             days), or 0 for the earliest time; the default is <end time> - 1h
-        end : string
-             (`tsdb` backend only) End (maximum) metric-sample time for the
+             days), or 0 for the earliest time; the default is an empty string
+             for when `end` is also not set - to delete the entire table - and
+             `0` when `end` is set
+        end : str
+             (`tsdb` backend only) End (maximum) data sample time for the
              delete operation, as a string containing an RFC 3339 time, a Unix
              timestamp in milliseconds, a relative time (`"now"` or
              `"now-[0-9]+[mhd]"`, where `m` = minutes, `h` = hours, and `'d'` =
-             days), or 0 for the earliest time; the default is "now"
-        if_missing : int
-            One of IGNORE or FAIL
+             days), or 0 for the earliest time; the default is an empty string
+             for when `start` is also not set - to delete the entire table -
+             and `0` when `start` is set
+        if_missing : int (frames_pb2 pb.ErrorOptions)
```

Comment:
Add this option to the ReadMe as well

Reply:
@talIguaz as I wrote in the first comment, a fuller doc update is being handled separately, as part of the new Frames reference in the platform doc, which is intended to replace the GitHub README. I've essentially stopped updating the README, I just didn't want it to contain errors until it's replaced, but I'm not adding to it.